src/RequestHandler/inprocess/inprocessapplication.cpp

```diff
@@ -292,46 +311,43 @@ IN_PROCESS_APPLICATION::SetStdOut(
                     // Periodically flush the log content to file
                     m_Timer.InitializeTimer(STTIMER::TimerCallback, &m_struLogFilePath, 3000, 3000);
                 }
+                else
+                {
+                    //
+                    // CreatePipe for outputting stderr to the windows event log.
+
+                    // Read pipe
+                    if (!CreatePipe(&hStdErrReadPipe, &hStdErrWritePipe, &saAttr, 0 /*nSize*/))
+                    {
+                        hr = HRESULT_FROM_WIN32(GetLastError());
```

Comment:
user did not ask for enabling log. this hr result is not consumed by other code path . You can remove this line